internal/connect/config.go

```diff
@@ -26,6 +26,15 @@ const (
 	defaultEnableSystemUptimeTracking = false
 )
 
+// Kinds of servers which are supported by SUSEConnect.
+type ServerType uint64
+
+const (
+	Unknown ServerType = iota
```

Comment:
Can we prefix this somehow? aka. `ApiProviderSCC` or `ApiProviderRMT` to avoid potential clashes?

Reply:
Done!